### PR TITLE
Multiplex gentx

### PR DESCRIPTION
**Pull request by Multiplex-Lab**

### CODE REVIEW

File: dydx-testnet-4/gentx/gentx-Multiplex.json [hunk 1]
```diff
@@ -0,0 +1 @@
...","website":"https://multiplexlabs.xyz/","security_contact":"","details":"Multiplex is on the mission to shape the future of Web3. It is a research-driven lab focusing on building the decentralized infrastructure."},"commission":{"rate":"1.000000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx1r5ygc8ekgldrcr97pkl2fj4uuuchsq3z42tdrp","validator_address":"dydxvaloper1r5ygc8ekgldrcr97pkl2fj4uuuchsq3zs5amgs","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"al/BQtjeoi4RXCdCBGaFIvvD9Ctm+8i1ylnzvy4dRNk="},"value":{"denom":"dv4tnt","amount":"50000000000"}}],"memo":"55b8e1fa0b6e8d746cbb84005fa53dc1a2b0f9c9@103.106.228.57:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AtncsSCbULeEVqCnIxzlA+Hw9Gok1Pjh7jiHxVpxphdT"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["XH7Hn2fUkeaGWSVBLIjR0F3mOvaZZDCRmzHEI0mr45t2T0jsqhA+ZKDymclcYkUoe91zOKbZzGW2H4rTHsN6Fg=="]}
```
Comment (2dpetkov) [inline dydx-testnet-4/gentx/gentx-Multiplex.json:1]: the [Prepare for Genesis](https://v4-teacher.vercel.app/validators/preparing_for_genesis) guide is a little different this time, the test token is `adv4tnt` and the self-delegation amount is `500000000000000000000000adv4tnt`

**Review by 2dpetkov — CHANGES_REQUESTED** [review]

File: dydx-testnet-4/gentx/gentx-Multiplex.json (hunk 1 above)
Comment (2dpetkov) [reply dydx-testnet-4/gentx/gentx-Multiplex.json:1]: @Multiplex-Lab kind reminder

**Review by Multiplex-Lab — COMMENTED** [review]
Fix

File: dydx-testnet-4/gentx/gentx-Multiplex.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Multuplex","identity":"","website":"","security_contact":"","details":""},"commission":{"rate":"1.000000000000000000","max_rate":"1.000000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx1r5ygc8ekgldrcr97pkl2fj4uuuchsq3z42tdrp","validator_address":"dydxvaloper1r5ygc8ekgldrcr97pkl2fj4uuuchsq3zs5amgs","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"al/BQtjeoi4RXCdCBGaFIvvD9Ctm+8i1ylnzvy4dRNk="},"value":{"denom":"adv4tnt","amount":"500000000000000000000000"}}],"memo":"55b8e1fa0b6e8d746cbb84005fa53dc1a2b0f9c9@103.106.228.57:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AtncsSCbULeEVqCnIxzlA+Hw9Gok1Pjh7jiHxVpxphdT"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["9QPu5wp55LkCKu/bOTI2PvyW8KXS3CA3VIoJX/ju195D3W6c47HgidyurAA577wpOGr5FFq5chegCKL66QuOfQ=="]}
```
Comment (2dpetkov) [inline dydx-testnet-4/gentx/gentx-Multiplex.json:1]: Please confirm if this is correct
`"moniker":"Multuplex"`

**Comment by Multiplex-Lab** [comment]
Fixd

**Comment by Multiplex-Lab** [comment]
Repair once again, thank you for your patience and understanding.

**Review by 2dpetkov — APPROVED** [review]

**Comment by 2dpetkov** [comment]
no worries!

**Pull request merged**